drivers/clock_control/clock_stm32f2_f4_f7.c

```diff
@@ -114,7 +114,16 @@ void config_plli2s(void)
 				       plli2sm(STM32_PLLI2S_M_DIVISOR),
 				       STM32_PLLI2S_N_MULTIPLIER,
 				       plli2sr(STM32_PLLI2S_R_DIVISOR));
+#if STM32_PLLI2S_Q_ENABLED
+	/* There is a Q divider on the PLLI2S to configure the PLL48CK */
+	LL_RCC_PLLI2S_ConfigDomain_48M(get_pll_source(),
+				       plli2sm(STM32_PLLI2S_M_DIVISOR),
+				       STM32_PLLI2S_N_MULTIPLIER,
+				       plli2sq(STM32_PLLI2S_Q_DIVISOR));
+#endif /* STM32_PLLI2S_Q_ENABLED */
```

Comment:
There is currently no configuration of main PLL Q channel. You should also add the LL_RCC_PLL_ConfigDomain_48M function in the config_pll_sysclock function.

Reply:
adding a LL_RCC_PLL_ConfigDomain_48M function in the config_pll_sysclock() function